crates/devices/virtio-console/src/console.rs

```diff
@@ -0,0 +1,105 @@
+use std::cmp;
+use std::io::{Write};
+use std::ops::Deref;
+use std::sync::{Arc, Mutex};
+use virtio_queue::{DescriptorChain};
+use vm_memory::{Bytes, GuestMemory};
+
+/// Console device resize feature.
+pub const VIRTIO_CONSOLE_F_SIZE: u64 = 0;
```

Comment:
Let's define these only when we are also implementing them.

Reply:
Done.